package.json

```diff
@@ -27,8 +27,8 @@
     "release": "np --no-release-draft"
   },
   "dependencies": {
-    "tsutils": "^3.0.0",
```

Comment:
This reordering is a result of running `npm install <JEST_PACKAGES>` (and would happen for any other npm install) - it's minor so we might as well rip this tiny bandaid off now